### PR TITLE
Dev

### LINKED ISSUE #916: Novelfull : Martial King’s Retired Life
**Describe the bug**
Hello

When i try to get the novel Martial King’s Retired Life, i have the error : 

` ! Error: 'src'
`
Thank you in advance

- Novel URL: https://novelfull.com/martial-kings-retired-life.html
- App Version: 2.26.3
- Package: pip
- OS: Windows 10

### PR DESCRIPTION
**Pull request by SirGryphin**
**Added:**
rpgnovels.com
dobelyuwai.wordpress
cclawtranslations

**Fixed:**
Fixed a few novel title, covers and authors. Also fixed on certain sources for blogs, fetching of none chapters links.

Also fixed #916

**Rejected List**
dsrealmtranslations.com - Site is down and has been for a while, also domain name is parked and no longer active.

### CODE REVIEW

File: lncrawl/sources/vistrans.py [hunk 1]
```diff
@@ -18,18 +18,22 @@ def read_novel_info(self):
...
+
+        # Removes none TOC links from bottom of page.
+        toc_parts = soup.select_one('.site-content')
+        for notoc in toc_parts.select('.sharedaddy, iframe'):
+            notoc.decompose()
 
         # Extract volume-wise chapter entries
-        # FIXME: Sometimes grabs social media link at bottom of page, No idea how to exclude links.
-        chapters = soup.select('table td [href*="vistranslations.wordpress.com"]')
+        # NOTE: Some chapters have two href links for each chapters so you get duplicate chapters.
```
Comment (dipu-bd) [inline lncrawl/sources/vistrans.py:35]: use `# TODO`

File: lncrawl/sources/88tang.py [hunk 2]
```diff
@@ -15,19 +15,19 @@ def read_novel_info(self):
...
         logger.info('Novel author: %s', self.novel_author)
 
+        # Removes none TOC links.
+        toc_parts = soup.select_one('.entry-content')
+        for notoc in toc_parts.select('.sharedaddy, .code-block, script, .adsbygoogle'):
+            notoc.decompose()
+
         # Extract volume-wise chapter entries
-        # FIXME: Sometimes grabs social media link at bottom of page, No idea how to exclude links.
         # FIXME: Chapter title are url links, it's the way translator formatted website.
```
Comment (dipu-bd) [inline lncrawl/sources/88tang.py:31]: use `#TODO: `

**Comment by dipu-bd** [comment]
I want to propose a change request here. Instead of messages like- `# FIXME` or `# NOTE`, can you use just `# TODO`? It helps better to find them later. Also, please follow this in the future. You can use `# TODO` when you want to leave something for later.

**Comment by SirGryphin** [comment]
I will make sure to use `# TODO` from now one, Also edited the ones in push request.

**Comment by dipu-bd** [comment]
thanks

**Pull request merged**